test/module/shared_model/cryptography/security_signatures_test.cpp

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "module/shared_model/builders/protobuf/test_block_builder.hpp"
+#include "module/shared_model/builders/protobuf/test_signature_builder.hpp"
+#include "module/shared_model/builders/protobuf/test_transaction_builder.hpp"
+
+/**
+ * @given Two signatures with same pub key but different signed
```

Comment:
signed -> signature

Reply:
In terms of sm_m objects Signature is a pair: PubKey × Signed. Here I want to reflect different signed explicitly.